### ISSUE TITLE
New premium upsell ad in social tab

### ISSUE THREAD

**Issue by allanrehhoff** [senlin/so-clean-up-wp-seo#85]
The YOAST Team has added a new upsell ad under the social tab.
It's not an area I suspect most people will ever use, but for someone (like me) that is very conservative about what they want to see in their backend, this is a big deal.
See attached screenshot.

## Expected behavior
A checkbox/setting to remove this new ad.
Or it to be included in the "Premium Upsell Admin Block" setting.

## Current behavior
Ads are showing, somewhat discrete, but an ad is an ad.

## Possible way to add this
These ads class starts with `SocialUpsell__PremiumInfoText` it could be fixed with:

```
div[class^="SocialUpsell__PremiumInfoText"] {
	display: none;
}
```

Not the prettiest, but it'll work.

## Screenshots
![2020-07-27_13-05-31](https://user-images.githubusercontent.com/24595705/88535962-358ef380-d00b-11ea-8153-236c4723877d.png)

Edit:
Wording and typos.

**Comment by senlin** [comment]
Thanks for your report @allanrehhoff I will try to add it this week and update this ticket accordingly.

**Comment by senlin** [comment]
released 3.14.1 many thanks @allanrehhoff !
